### PR TITLE
Enhance synchronization of events to enhance multitab experience [OOB]

### PR DESCRIPTION
**Pull request by elopezanaya**
### Description
_Include a description of the problem to be solved_

## Solution Proposed
_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

### Acceptance criteria
_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

### Sanity Tests
-   [ ] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

File: chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx [hunk 1]
```diff
@@ -671,14 +695,24 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
...
+        const inMemoryState = executeReducer(state, { type: LiveChatWidgetActionType.GET_IN_MEMORY_STATE, payload: null });
 
+        /**
+         * Trick to prevent if the current tab has a different state that the future state to use the future state.
+         * 
+         * This is to prevent when a different tab updates conversation id, then a second tab will react to save the new value with a mix of old values, 
+         * the minimize state is important because is used to compute visibility of components, 
+         */
+        if (state.appStates.isMinimized !== inMemoryState.appStates.isMinimized) {
+            state.appStates.isMinimized = inMemoryState.appStates.isMinimized;
```
[the remark this answers is not in the document]
Comment (elopezanaya) [reply chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx:707]: this inline swap is just for local storage and its using future value of the state, so there is no harm.

**Review by Erli-ms — APPROVED** [review]
